test/brod_consumer_SUITE.erl

```diff
@@ -362,6 +363,22 @@ t_direct_fetch(Config) when is_list(Config) ->
   ?assertEqual(Key, Msg#kafka_message.key),
   ok.
 
+t_direct_fetch_with_connect_timeout(Config) when is_list(Config) ->
```

Comment:
one last thing
this test case is duplicating the above one.
Maybe rename it to `t_resolve_offset` and no need to test anything else than the call to `brod:resolve_offset/6`.
e.g.
```
t_resolve_offset(Config) when is_list(Config) ->
  Client = ?config(client),
  Topic = ?TOPIC,
  Partition = 0,
  {ok, Offset} = brod:resolve_offset(?HOSTS, Topic, Partition,
                                     ?OFFSET_LATEST, ?config(client_config),
                                     #{timeout => timer:seconds(45)}),
  ?assertTrue(is_integer(Offset)).
```